common/changes/@itwin/appui-react/MBudrevicius-UpdateDocumentation_2024-09-30-12-32.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/appui-react",
+      "comment": "Updated documentation",
```

I'd leave these empty, README update is a somewhat internal change, consumers don't need to care about that when upgrading
```suggestion
      "comment": "",
```